content/blog/ways-to-interact-with-kubernetes-clusters-managed-by-hpe-ezmeral-container-platform.md

```diff
@@ -0,0 +1,180 @@
+
+HPE Ezmeral Software Platform consists of a bundle of software that helps you run, manage, control, and secure the apps, data, and IT that run your business. A major component is the HPE Ezmeral Container Platform (HPE ECP),a unified cloud container software platform built on Kubernetes (K8s). You can use it to deploy a new Kubernetes cluster with a few clicks. But how exactly do you connect through HPE ECP to interact with K8s? Don't panic! This blog post will introduce to you most, if not all, of the ways to connect with the Kubernetes using HPE ECP.
+
+
+
+
+## WebUI
+The first method by which you can interact with a Kubernetes Clusters managed by HPE ECP is, of course, through the web user interface (WebUI).
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Cluster.png)     | In the Main Menu, if you are the administrator of the HPE Ezmeral CP, you can manage and monitor the status of the K8s clusters managed by HPE ECP.  |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Dashboard.png)   | You can access the Kubernetes Dashboard just as you would do with open source Kubernetes.  |
+
+## WebTerminal
+Inside the Kubernetes Tenant, at the bottom, there is a web terminal for you to interact with Kubernetes using the ```kubectl``` command.
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant.png)      | At the bottom, you can click "Initialize" to initiate the web terminal instance.  |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant-02.png)   | Type your ``` kubectl ``` command to interact with Kubernetes.  |
+
+
+## HPE Kubectl Plugin
+If you want to use kubectl remotely, you must first install the kubectl-plugin to get the session ID to access HPE Ezmeral. To install the kubectl-hpecp plugin, you can run the following command. The HPE kubectl plugin is required to establish the tenant’s authenticated kubectl requests. Therefore, if you want to use kubectl remotely as a tenant user, you must first install the HPE kubectl plugin (kubectl-hpecp) to get an HPE ECP authentication token (a session ID), fetch the kubconfig manifest file, and then interact with the managed K8s clusters.
+
+To install the kubectl-hpecp plugin, first fetch the plugin from the WebUI. Log in as a tenant user, and click on Download HPE Kubectl Plugin (as shown in the picture below) and download the plugin and installation instructions according to your target operating system. The following codes are executed in Linux environment.
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant-03.png)      | You can download the required binary here as well.  |
+
+### Installation of ```kubectl``` and ```kubectl-hpecp```
+#### Step 1: Make sure you have ```kubectl``` install.

```

Comment:
Was just going to comment on the same thing as Denis... "install" should be "installed".